solidity/test/VendingMachineTest.js

```diff
@@ -130,6 +134,43 @@ describe("VendingMachine", async function() {
         "ERC721: transfer caller is not owner nor approved.",
       )
     })
+
+    it("updates the minted supply", async () => {
+      const mintedSupply = await vendingMachine.getMintedSupply()
+
+      await tbtcDepositToken.forceMint(owner, tdtId)
+      await tbtcDepositToken.approve(vendingMachine.address, tdtId, {
+        from: owner,
+      })
+      await vendingMachine.tdtToTbtc(tdtId, {from: owner})
```

Comment:
Feels like a strangely grey box way to test this. Why not just force some TBTC minting and ensure the VM reports it?

Reply:
I was trying to cover a realistic code path, but if there's a reason to prefer `TBTCToken.frozenMint()` I'm open to it

Reply:
Mostly it's that all tests here are white-box and do force minting, so doing something completely different makes it seem like there's a underlying reason why this wouldn't work the way the other tests would. Not a huge deal.